src/Integration/ProjectToLanguageMapper.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace SonarLint.VisualStudio.Integration
+{
+    public static class ProjectToLanguageMapper
+    {
+        internal static readonly IDictionary<Guid, Language> KnownProjectTypes;
+
+        static ProjectToLanguageMapper()
+        {
+            KnownProjectTypes = new Dictionary<Guid, Language>();
+            KnownProjectTypes[new Guid(ProjectSystemHelper.CSharpProjectKind)] = Language.CSharp;
+            KnownProjectTypes[new Guid(ProjectSystemHelper.VbProjectKind)] = Language.VBNET;
+            // TODO: handle core project types
+//           KnownProjectTypes[new Guid(ProjectSystemHelper.CSharpCoreProjectKind)] = Language.CSharp;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS125)